### PR TITLE
ListView + CarouselView + CollectionView

### PR DESCRIPTION
**Pull request by edgarfgp**

### CODE REVIEW

File: docs/content/en/v2/api/controls/carousel-view.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+id: "carousel-view"
```
Comment (TimLariviere) [inline docs/content/en/v2/api/controls/carousel-view.md:2]: Can you prefix all ids with `v2-` please?
Currently a limitation of the website. I would prefer only showing v2 docs, but it's mixing v1 and v2 for now

File: docs/content/en/v2/api/controls/collection-view.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    (fun item -> Label($"{item}")))
+    .selectionMode(SelectionMode.Single)
+    .onItemTapped(ItemTapped)
+    
+type Group(headerData: string, footerData: string, items: IEnumerable<int>) =
+    inherit ObservableCollection<int>(items)
+    member _.HeaderData = headerData
+    member _.FooterData = footerData
+
+GroupedCollectionView(items)
```
Comment (TimLariviere) [inline docs/content/en/v2/api/controls/collection-view.md:64]: Missing the groups creation

**Review by TimLariviere — APPROVED** [review]
Looking good 👍 
Just a few comments

File: docs/content/en/v2/api/controls/carousel-view.md (hunk 1 above)
Comment (edgarfgp) [reply docs/content/en/v2/api/controls/carousel-view.md]: Fixed :)

File: docs/content/en/v2/api/controls/collection-view.md (hunk 2 above)
Comment (edgarfgp) [reply docs/content/en/v2/api/controls/collection-view.md]: Updated

**Pull request merged**